osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyAttributes.cs

```diff
@@ -1,21 +1,76 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
 using osu.Game.Rulesets.Difficulty;
+using osu.Game.Rulesets.Mods;
 
 namespace osu.Game.Rulesets.Osu.Difficulty
 {
     public class OsuDifficultyAttributes : DifficultyAttributes
     {
+        [JsonProperty("aim_strain")]
         public double AimStrain { get; set; }
+
+        [JsonProperty("speed_strain")]
         public double SpeedStrain { get; set; }
+
+        [JsonProperty("flashlight_rating")]
         public double FlashlightRating { get; set; }
+
+        [JsonProperty("slider_factor")]
         public double SliderFactor { get; set; }
+
+        [JsonProperty("approach_rate")]
         public double ApproachRate { get; set; }
+
+        [JsonProperty("overall_difficulty")]
         public double OverallDifficulty { get; set; }
+
         public double DrainRate { get; set; }
+
         public int HitCircleCount { get; set; }
+
         public int SliderCount { get; set; }
+
         public int SpinnerCount { get; set; }
+
+        public override IEnumerable<(int attributeId, object value)> ToDatabaseAttributes()
+        {
+            foreach (var v in base.ToDatabaseAttributes())
+                yield return v;
+
+            yield return (1, AimStrain);
```

Comment:
Think this PR would be a good opportunity to make `const int`s for each of these? Would definitely help set my mind at ease.

Reply:
I think raw numbers may be better until the TODOs about attributes being stored under wrong indices are fixed, because otherwise those cases are going to read really badly. But maybe with a comment it is fine either way?

Reply:
I think those cases can be left as raw `int`s alonside the TODO?

Reply:
I don't see those being fixed as a priority, since there's just too many moving parts and we're changing score storage/pp processor. Do agree with making them `const int`s.